hack/verify-e2e-test-ownership.sh

```diff
@@ -16,9 +16,7 @@
 
 # This script verifies the following e2e test ownership policies
 # - tests MUST start with [sig-foo]
-# - tests MUST use top-level SIGDescribe
 # - tests SHOULD NOT have multiple [sig-foo] tags
-# - tests MUST NOT use nested SIGDescribe
```

Comment:
In other words, verify-e2e-test-ownership.sh is now a bit less precise?

I'm fine with that to get us going with ginkgo/v2, but it might be worth to create an enhancement issue for restoring this functionality somehow.

Reply:
> verify-e2e-test-ownership.sh is now a bit less precise?

this script will verify what it has in the test spec, I don't think this could be more precise unless something changed in Ginkgo, anyway, I create an issue here to track this,

create an issue here: https://github.com/kubernetes/kubernetes/issues/109724